src/easylink/step.py

```diff
@@ -1,15 +1,28 @@
 from abc import ABC, abstractmethod
+from dataclasses import dataclass
 from pathlib import Path
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Callable, Dict, List, Optional, Tuple
 
-from networkx import MultiDiGraph
+import networkx as nx
+from layered_config_tree import LayeredConfigTree
 
 from easylink.implementation import Implementation
+from easylink.utilities import paths
+from easylink.utilities.data_utils import load_yaml
 
 if TYPE_CHECKING:
     from easylink.configuration import Config
 
 
+@dataclass
+class InputSlot:
```

Comment:
Why not use this class in pipeline_schema_constants.py?

Reply:
yeah, you're probably right